packages/resource-deployment/templates/on-demand-scan-schedule.template.json

```diff
@@ -10,8 +10,8 @@
             "maxTaskRetryCount": 0
         },
         "jobManagerTask": {
-            "id": "url-web-api-scan-job-manager-task",
-            "commandLine": "/bin/bash -c run-web-api-scan-job-manager.sh",
+            "id": "on-demand-url-scan-job-manager-task",
+            "commandLine": "/bin/bash -c run-on-demand-scan-job-manager.sh",
```

Comment:
did you make sure this is gonna work, did we change the ".sh" file name

Reply:
i changed the sh file too.